README.rst

```diff
@@ -25,6 +25,8 @@ Install the dependencies::
 
     pip install -r requirements.txt
 
+**Note:** You might need to run ``brew install postgresql@14`` to install PostgreSQL 14 (or a similar command for your package manager) to install PostgreSQL® as a ``psycopg2`` build dependency. 
```

Comment:
```suggestion
**Note:** If that fails because it can't install ``psycopg2-binary``, run ``brew install postgresql@14`` to install PostgreSQL 14 (or a similar command for your package manager) and then try again.
```
I think it's better to put the problem first, and also I believe it's specifically a problem with the binary package.